libs/fluxc/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
     alias(sharedLibs.plugins.kotlin.android)
     alias(sharedLibs.plugins.kotlin.parcelize)
     alias(sharedLibs.plugins.kotlin.kapt)
```

Comment:
Suggestion (💡): Since the `kotlin` version is now aligned (a13f97ee87ae671366300ed7ff5183cf68bbfd08), consider replacing `sharedLibs` with `libs` for everything `plugins.kotlin`, at least on the `plugins` side of things here.

PS: `android.library` is already matching so that can be replaced too, which leaves us with only `dependencies` on `sharedLibs` (for later).

Reply:
FYI: This is done as part of #21353 and 88a675f760b3baa8e10d173c339a040fdf209c87.